tests/basic/basic_test.go

```diff
@@ -301,6 +301,7 @@ var _ = Describe("{AppScaleUpAndDown}", func() {
 })
 
 var _ = AfterSuite(func() {
+	CheckForCoreFiles()
```

Comment:
Add `CheckForCoreFiles` in `AfterSuite` of other tests too (tests/reboot, tests/volume_ops, tests/upgrade, tests/drive_failure)